ql/src/java/org/apache/hadoop/hive/ql/io/parquet/convert/ETypeConverter.java

```diff
@@ -707,6 +707,10 @@ protected HiveDecimalWritable convert(Binary binary) {
                   (DecimalTypeInfo) hiveTypeInfo);
             }
 
+            @Override
+            public boolean hasDictionarySupport() {
```

Comment:
please create a follow-up ticket for supporting dictionary and make it visible by a code comment here

Reply:
Followup ticket has been created and added TODO comment for ref
https://issues.apache.org/jira/browse/HIVE-27529